cg/server/api.py

```diff
@@ -216,11 +222,25 @@ def parse_case(case_id):
     return jsonify(**case.to_dict(links=True, analyses=True))
 
 
+@BLUEPRINT.route("/cases/delivery_message", methods=["GET"])
+def get_cases_delivery_message():
+    delivery_message_request: DeliveryMessageRequest = DeliveryMessageRequest.model_validate(
+        request.args
+    )
+    service = DeliveryMessageService(db)
+    try:
+        response: DeliveryMessageResponse = service.get_delivery_message(delivery_message_request)
+        return jsonify(response.model_dump()), HTTPStatus.OK
+    except (CaseNotFoundError, OrderMismatchError) as error:
+        return jsonify({"error": str(error)}), HTTPStatus.BAD_REQUEST
+
+
 @BLUEPRINT.route("/cases/<case_id>/delivery_message", methods=["GET"])
 def get_case_delivery_message(case_id: str):
     service = DeliveryMessageService(db)
```

Comment:
If you extract the service to the `ext` module, this instantiation is unnecessary

Reply:
Moved it!